crates/walrus-service/bin/client.rs

```diff
@@ -61,7 +61,12 @@ fn client() -> Result<()> {
     let mut app = ClientArgs::parse().inner;
     app.extract_json_command()?;
 
-    tracing::info!("client version: {VERSION}");
+    if !env::var("INTERNAL_RUN")
+        .map(|v| v == "true")
+        .unwrap_or(false)
```

Comment:
Nit: can we extract this to a global function? This condition has been used in multiple places.